content/zh/docs/hertz/tutorials/basic-feature/middleware/recovery.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Recovery"
+date: 2022-12-15
+weight: 2
+description: >
+
+---
+
+Recovery中间件是Hertz框架预置的中间件，使用 `server.Default()` 可以默认注册该中间件，为Hertz框架提供panic恢复的功能。
```

Comment:
What do I do if I don't want to use Hertz.Default? Can you add to it?

Reply:
done, thx~